pkg/cmd/audit/io.go

```diff
@@ -302,3 +303,54 @@ func GetEvents(auditFilename string) ([]*auditv1.Event, error) {
 
 	return ret, err
 }
+
+func PrintTopByHTTPStatusCodeAuditEvents(writer io.Writer, events []*auditv1.Event) {
+	countHTTPStatusCode := map[int32][]*auditv1.Event{}
+
+	for _, event := range events {
+		if event.ResponseStatus == nil {
```

Comment:
@deads2k  are we not including the event with `stage: RequestReceived` where `event.ResponseStatus` is always nil, or is it being filtered up in the call chain?

Reply:
> @deads2k are we not including the event with `stage: RequestReceived` where `event.ResponseStatus` is always nil, or is it being filtered out up in the call chain?

Not yet.  You should make a PR :)